src/Codeception/Module/Doctrine2.php

```diff
@@ -77,6 +77,14 @@
  * You cannot use `cleanup: true` in an acceptance test, since Codeception and your app (i.e. browser) are using two
  * different connections to the database, so Codeception can't wrap changes made by the app into a transaction.
  *
+ * Change purge mode doctrine fixtures:
```

Comment:
Please improve this sentence.
I think that it should be `Change purge mode of doctrine fixtures`.

A short description of why it could be necessary to change purge mode would be useful too.

Reply:
@Naktibalda Just noticed your comment here, after I just changed this line to what is not really a description, but better than nothing ;-) https://github.com/Codeception/module-doctrine2/pull/39